From 110dfd10e9c5c91c2578032aa5e1c5a778b944f3 Mon Sep 17 00:00:00 2001
From: lijian <jian.li@freescale.com>
Date: Thu, 29 Dec 2011 06:12:57 +0800
Subject: [PATCH 1/3] ENGR00133115 Add .ts,.vob,.f4v,.mkv,.webm mimetype

Add these mimetype to make these files to http
streaming not download source.

Signed-off-by: lijian <jian.li@freescale.com>

Conflicts:
	luni/src/main/java/libcore/net/MimeUtils.java
---
 luni/src/main/java/libcore/net/MimeUtils.java | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/luni/src/main/java/libcore/net/MimeUtils.java b/luni/src/main/java/libcore/net/MimeUtils.java
index e36b3d1..68ce110 100644
--- a/luni/src/main/java/libcore/net/MimeUtils.java
+++ b/luni/src/main/java/libcore/net/MimeUtils.java
@@ -377,6 +377,12 @@ public final class MimeUtils {
         add("video/x-webex", "wrf");
         add("x-conference/x-cooltalk", "ice");
         add("x-epoc/x-sisx-app", "sisx");
+        add("video/mpeg", "vob");
+        add("video/mpeg", "ts");
+        add("video/x-flv", "f4v");
+        add("video/x-matroska", "mkv");
+        add("video/x-matroska", "webm");
+        applyOverrides();
     }
 
     private static void add(String mimeType, String extension) {
-- 
1.8.0

